admin/syslog-ng/Makefile

```diff
@@ -54,7 +54,8 @@ CONFIGURE_ARGS += \
          --disable-linux-caps \
 	 --disable-smtp \
 	 --disable-redis \
-         --enable-prce \
+	 --enable-prce \
+	 --disable-tests \
```

Comment:
Stray `\` ?

Reply:
No, was already like that, terminating on the empty line.